tools/windows-node-installer/pkg/cloudprovider/aws/ec2.go

```diff
@@ -570,7 +570,7 @@ func (a *AwsProvider) verifyAndUpdateSg(myIP string, sg *ec2.SecurityGroup, vpc
 func (a *AwsProvider) GetRulesForSgUpdate(myIP string, rules []*ec2.IpPermission, vpcCidr string) []*ec2.IpPermission {
 	rulesForUpdate := make([]*ec2.IpPermission, 0)
 
-	ports, hasClusterCidrRule := examineRulesInSg(myIP, rules, vpcCidr)
+	ports, hasClusterCidrRule := ExamineRulesInSg(myIP, rules, vpcCidr)
```

Comment:
nit: The fact we're making functions public to make sure we're testing properly bothers me

Reply:
I thought exporting functions were only required for e2e tests. Is this needed for unit tests too?

Reply:
It isn't required here, they work fine without exporting. I can remove exports.